### ISSUE TITLE
Settings schema Error! with logseq-awesome-links

### ISSUE THREAD

**Issue by maxtmax** [yoyurec/logseq-awesome-links#61]
**Info:**
- OS: Windows
- Logseq ver: 0.10
- Plugin ver: v1.15.16

**Description:**
After an upgrade of logseq to version 0.10 the plugin Awesome Links stopped working properly.
When clicking the settings icon I get the following error: **Settings schema Error!**

**To Reproduce:**

1. Go to settings
2. Go to plugin settings

**Expected behavior:**
It should show all the plugins installed and let you configure them.

**Screenshots**
![image](https://github.com/yoyurec/logseq-awesome-links/assets/131976388/06ec2d68-7ec1-478f-ac6f-e0837f215ca9)

**Comment by maxtmax** [comment]
The following may or may not be related to the above-mentioned issue but in general, the overall functionality of the plugin stopped working in general with the following observations:

- The icons and colours that are configured in the properties for each page do not show up in the references.
- When I try to uninstall or disable the plugin, I cannot do so [have to resort to physically deleting the folder on the hard drive].

**Comment by maxtmax** [comment]
I figured out the problem and consequently fixed it, here are the steps that I followed to resolve the issue:

1. Go to: C:\Users\<username>\.logseq\settings\logseq-awesome-links.json
2. Observe that the file contains a collection of NULL values as seen in the screenshot below
3. Replace the content of the file with {}
4. Re-load logseq: the file will be rebuilt to contain proper settings
5. All functionality will be restored afterwards

![image](https://github.com/yoyurec/logseq-awesome-links/assets/131976388/1ae8d2f3-69db-4abf-9a77-cc434d5dbdc8)
